Sofa/Component/Constraint/Projective/src/sofa/component/constraint/projective/LinearVelocityProjectiveConstraint.inl

```diff
@@ -311,6 +314,85 @@ void LinearVelocityProjectiveConstraint<TDataTypes>::projectJacobianMatrix(const
 
 }
 
+template <class DataTypes>
+void LinearVelocityProjectiveConstraint<DataTypes>::applyConstraint(const core::MechanicalParams* mparams, linearalgebra::BaseVector* vector, const sofa::core::behavior::MultiMatrixAccessor* matrix)
+{
+    SOFA_UNUSED(mparams);
```

Comment:
Shouldn't you use `d_continueAfterEnd` and/or `finished` here? If `d_continueAfterEnd` is false and `finished` is true, should the matrix projection be applied?